prov/psm3/configure.m4

```diff
@@ -66,16 +66,15 @@ ifelse('
 
 		AC_MSG_CHECKING([for -msse4.2 support])
 		save_CFLAGS=$CFLAGS
-		CFLAGS="$CFLAGS -msse4.2"
+		CFLAGS="$CFLAGS -msse4.2 -O0"
 		AC_LINK_IFELSE(
 			[AC_LANG_PROGRAM(
-				[#include <nmmintrin.h>],
-				[unsigned int crc = 0;
-				 crc = _mm_crc32_u32(crc, 0);
-				 return crc == 0;])
+				[[#include <nmmintrin.h>]],
+				[[unsigned int crc = 0;
+				  crc = _mm_crc32_u32(crc, 0);
+				  return crc == 0;]])
 			],[
 				AC_MSG_RESULT([yes])
-				psm3_crc_happy=1
```

Comment:
If this is a PSM3-specific set of flags, please don't use the generic name `ARCH_CFLAGS` -- please at least put a `PSM3_` or `psm3_` prefix on it (since these shell variables are in the global scope).

Reply:
Agree will change